From f666fcad783464b03289ba66b7ddf672b0c2b512 Mon Sep 17 00:00:00 2001
From: Shammah Chancellor <shammah.chancellor@gmail.com>
Date: Sun, 20 Jun 2021 15:26:04 -0700
Subject: [PATCH 129/196] Update nonces for CreateNewBlock_validity test (#207)

This commit updates the nonces for the CreateNewBlock_validity test
which became broken after the update of the mainnet genesis block.
---
 src/test/miner_tests.cpp | 44 ++++++++++++++++++++--------------------
 1 file changed, 22 insertions(+), 22 deletions(-)

diff --git a/src/test/miner_tests.cpp b/src/test/miner_tests.cpp
index 61d3efd9f..a455feb20 100644
--- a/src/test/miner_tests.cpp
+++ b/src/test/miner_tests.cpp
@@ -65,28 +65,28 @@ BlockAssembler MinerTestingSetup::AssemblerForTest(const Config &config) {
 }
 
 constexpr uint64_t blockinfo[] = {
-    138489447073, 2000123001,   35192735278,  30743516887,  90270421301,
-    4746156542,   269517074003, 50530474709,  201348804349, 131694823915,
-    96838508997,  54292234081,  146777944833, 194579551989, 17459672428,
-    239463286493, 35972158971,  131229981500, 288173081779, 8271781121,
-    14054617812,  1168893791,   67365635174,  196656931529, 35800309163,
-    17851574064,  26749261103,  14953015794,  22247598424,  16778982303,
-    119328830052, 19182627651,  133744171463, 161386696561, 5583120677,
-    37193104054,  74777620295,  240886925522, 202169284734, 37158187173,
-    180572907622, 34681770166,  218161288478, 5698624760,   123626801781,
-    8494330929,   134180517818, 380634240428, 1112112668,   19262695841,
-    727120508,    65255482113,  90905170725,  45882969013,  112934652766,
-    15470497544,  89665853312,  4833940537,   70579955079,  131318555366,
-    51087976648,  18737029675,  112319798991, 154404109467, 164492077002,
-    26800008678,  8462192695,   35955108190,  70609431501,  141973644757,
-    171937771431, 27571969114,  175359612626, 9514202569,   144770109340,
-    18542728507,  67998505071,  274075528823, 202967048402, 18535319865,
-    937791320,    60483413889,  43840334254,  38996608745,  135126199815,
-    136500720625, 3691630528,   85448072763,  75537584241,  205777394006,
-    22255213872,  170642741506, 130688629769, 219159895699, 97098261172,
-    118269396602, 87328266489,  185378645008, 10623915215,  7452183400,
-    14890598267,  671249038,    97235700784,  35390245092,  144350615945,
-    67256243540,  71934634370,  12275123663,  151449715016, 8967451371,
+    42203852168,  43038628490,  47843844182,  312644427462, 45556063700,
+    25901875104,  55021537363,  1638082531,   6643956068,   52561954928,
+    179780982351, 77589891933,  29402077606,  83334507699,  49852671317,
+    9608565622,   158596606919, 1172556629,   7721188638,   63460163196,
+    1202106896,   26067947412,  2999839922,   271723472214, 48839718296,
+    125629738914, 60407801998,  44075529770,  203965062525, 283238928211,
+    270848817254, 197911958077, 10767242253,  77647693684,  70479470827,
+    108105072607, 272390183583, 16469615266,  66876012552,  30440231343,
+    31553846823,  20177791231,  40423375076,  62798795989,  60947383364,
+    55085994073,  24307333324,  123766466613, 306453796685, 119526571779,
+    72270492825,  225627844887, 17063869771,  3774082791,   17464618962,
+    6623615073,   17579233408,  9804517381,   56376386440,  57101496844,
+    60927545107,  4579131090,   89443877872,  58923008710,  59505931283,
+    99040523933,  24089767168,  25999089378,  151454685746, 4951604216,
+    4754953592,   15653058348,  398626391240, 25982207711,  227341538536,
+    21093303073,  23088703428,  32837844416,  11436218059,  21515047340,
+    11215339413,  64537999671,  72921808394,  39474990225,  35700646414,
+    99423627098,  36198182295,  11930833411,  39959119058,  160087458722,
+    37864978644,  38226830380,  35792333014,  2367678124,   32578862003,
+    91243218276,  51322504579,  68276010850,  45269927836,  270817185932,
+    16421867139,  44039973372,  61460641948,  23505474200,  62441820551,
+    80202485781,  13192058976,  75436666866,  15180626918,  11739014012,
 };
 
 // In Logos, half of the block subsidy goes to the miner.
-- 
2.39.5 (Apple Git-154)

